### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mcmillanator**
Still have work to do on this one.

TODO: 
- [x] add CSV
- [x] complete my case method

### CODE REVIEW

**Comment by rpearce** [comment]
Would you mind reducing the indentations down to 2 spaces in the future?

File: classes/User.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require 'csv'
+class User
+
+	attr_accessor :name, :pin, :balance, :authenticated
+
+	def authenticate(name, pin)
+		@name = name.capitalize
+		@pin = pin.to_i
+    CSV.foreach("users.csv", headers: true) do |row|
```
Comment (rpearce) [inline classes/User.rb]: Nitpick: The spacing here is a bit inconsistent

**Comment by rpearce** [comment]
We went over most of this in class. If you want me to add more commentary, then let me know. Otherwise, :+1:

**Comment by mcmillanator** [comment]
Fixed the indention.

**Pull request merged**